packages/material-ui-codemod/src/v5.0.0/jss-to-styled.test/tenth.expected.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { styled } from '@material-ui/core/styles';
+const PREFIX = 'tenth';
+
+const classes = {
+  header: `${PREFIX}-header`,
+  img: `${PREFIX}-img`
+};
+
+// TODO jss-to-styled codemod: The Fragment root was replaced by div. Change the tag if needed.
+const Root = styled('div')((
+  {
+    theme
+  }
+) => ({
+  [`& .${classes.header}`]: {
+    marginLeft: theme.spacing(5),
+    paddingRight: theme.spacing(3),
+    marginRight: 'auto',
+    minWidth: 400,
+  },
+
+  [`& .${classes.img}`]: {
+    marginTop: theme.spacing(4),
+  }
+}));
+
+export default function Page() {
+
+
+  return (
+    (<Root>
```

Comment:
Where did React.Suspense go? We still need it, no?

Reply:
What do you mean? The Suspense is in the twelfth example. I am not transforming anything if Suspense is the root, we may add this support later if someone asks for it.

Reply:
The confusion comes from not having `twelfth.expected`. Why is there no file with the transformation result?

Reply:
If the Root is React.Suspense I am not transforming the file. We can add the support for it later if people ask for it.

Reply:
@mnajdova Ok perfect, Sebastian got the root cause.